GVFS/GVFS.Platform.Windows/WindowsPlatform.cs

```diff
@@ -352,7 +364,7 @@ public override bool IsConsoleOutputRedirectedToFile()
 
         public override bool IsGitStatusCacheSupported()
         {
-            return File.Exists(Path.Combine(GVFSPlatform.Instance.GetDataRootForGVFSComponent(GVFSConstants.Service.ServiceName), GVFSConstants.GitStatusCache.EnableGitStatusCacheTokenFile));
+            return File.Exists(Path.Combine(GVFSPlatform.Instance.GetSecureDataRootForGVFSComponent(GVFSConstants.Service.ServiceName), GVFSConstants.GitStatusCache.EnableGitStatusCacheTokenFile));

```

Comment:
Does this change where we are looking for the status cache token file?  Will this turn of status cache?

Reply:
As discussed [added ](https://github.com/microsoft/VFSForGit/pull/1597/commits/84d4016e3a2216d66579045eba8a72d9b7521699)a migration for status cache token file.